src/java/boa/functions/BoaAstIntrinsics.java

```diff
@@ -209,9 +210,12 @@ private static void openMap() {
 		final Configuration conf = new Configuration();
 		try {
 			final FileSystem fs = FileSystem.get(conf);
-			final Path p = new Path("hdfs://boa-njt/",
```

Comment:
My last comment still stands - all 3 paths are using a single `DefaultProperties.localDataPath` variable, which is wrong.